src/overlays/ovl_i1/DFEE30.c

```diff
@@ -1,9 +1,27 @@
 #include "global.h"
 #include "DFEE30.h"
 
+extern s32 D_i1_8019C0B8;
+
 #pragma GLOBAL_ASM("asm/us/nonmatchings/overlays/ovl_i1/DFEE30/func_i1_801920F0.s")
 
-#pragma GLOBAL_ASM("asm/us/nonmatchings/overlays/ovl_i1/DFEE30/func_i1_801922DC.s")
+void func_i1_801922DC(Boss* this) {
+    Boss* boss = gBosses;
+    s32 i;
+
+    for (i = 0; i < 4; i++) {
```

Comment:
does this match? remember to remove the boss++ below
```suggestion
for (i = 0; i < 4; i++, boss++) {